code/modules/projectiles/gun_system.dm

```diff
@@ -198,10 +198,26 @@
 	var/recoil_backtime_multiplier = 2
 	///this is how much deviation the gun recoil can have, recoil pushes the screen towards the reverse angle you shot + some deviation which this is the max.
 	var/recoil_deviation = 22.5
-	///How much the bullet scatters when fired while wielded.
+	///How much the bullet currently scattered when last fired.
 	var/scatter	= 4
 	///How much the bullet scatters when fired while unwielded.
 	var/scatter_unwielded = 12
+	///Maximum scatter
+	var/max_scatter = 360
+	///Maximum scatter when wielded
+	var/max_scatter_unwielded = 360
```

Comment:
I'm assuming this is basically just placeholder, but the default should still be something somewhat reasonable (i.e. less than 180)

Reply:
doenst matter much, it is that high just so it doenst care or mess with any gun that doenst have the system implemented